inception-project/inception
# #348 - deleting features

**What's in the PR**
- Refactored DB so that "feature" field of the Recommender becomes a foreign key reference to the AnnotationFeature table
- Placed a cascade-on-delete constraint which deletes the recommender when the corresponding feature is deleted
- The DB refactoring intentionally does not delete the deprecated column - this column should be dropped by a separate changeset in a later version/revision

**How to test manually**
* Create a custom span layer with a custom string feature
* Create a recommender for that that feature
* Delete the feature
* Check if the recommender is still there

**Automatic testing**

**Documentation**
